hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/MockHoodieHiveSyncTool1.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.hudi.hive;
+
+import org.apache.hudi.sync.common.HoodieSyncTool;
+
+import org.apache.hadoop.conf.Configuration;
+
+import java.util.Properties;
+
+public class MockHoodieHiveSyncTool1 extends HoodieSyncTool {
+
+  public static boolean syncSuccess;
+
+  public MockHoodieHiveSyncTool1(Properties props, Configuration hadoopConf) {
```

Comment:
I did not mean to suggest mock this way. sorry. 
I was meaning to suggest mock using mockito lib. 
can you check TestCallbackHttpClient for examples.